### ISSUE TITLE
Feature Phone UI

### ISSUE THREAD

**Issue by ollieparanoid** [postmarketOS/pmbootstrap#944]
(This seems to be a recurring topic in the channel, so I'm writing it down in an issue.)

There was this idea floating around of making a feature phone UI, that makes any touchscreen smartphone look and behave like one of these old nokia phones. Possibly with extra big buttons and text.

It could be useful for old people, who are used to such devices and don't want to learn a touch screen based UI. And in general it wouldn't eat up much resources.

If someone is interested in this, go ahead and make something! UI could be based on libweston for example, that way it would work with all devices where weston is already working.

**Comment by joshas** [comment]
Symbian is open-source https://github.com/SymbianSource , could be a good source of inspiration.

**Comment by PureTryOut** [comment]
Doesn't Symbian just use Hildon? That's already packaged, but it's more a smartphone UI than a featurephone UI.

**Comment by JBBgameich** [comment]
Afaik Hildon is from Maemo, not from Symbian.

**Comment by BluRaf** [comment]
There was Qtopia platform used in [Greenphone](https://en.wikipedia.org/wiki/Greenphone) and some Motorola featurephones, now maintained as [QtMoko](http://qtmoko.sourceforge.net/).

**Comment by ollieparanoid** [comment]
Yeah Symbian and Hildon are entirely different. Nokia made phones with both, that's the connection.
Symbian has its own [kernel](https://en.wikipedia.org/wiki/Symbian#Symbian_kernel) while Hildon is the UI for Maemo (which is a Debian fork, using the Linux kernel).

**Comment by PureTryOut** [comment]
I've send an email to the QtMoko mailing list, asking for assistance in porting it's UI to postmarketOS. From a quick look around their website, I couldn't find any standalone package sources, but maybe I'm just completely blind. So far, QtMoko seems really promising though!

**Comment by BluRaf** [comment]
@PureTryOut There's also a fork maintained by OpenPhoenux/Tinkerphones community: http://projects.goldelico.com/p/gta04-qtmoko/ with some discussion on this mailing list: http://lists.goldelico.com/pipermail/community/2017-October/thread.html
